eva-accession-ws/src/main/java/uk/ac/ebi/eva/accession/ws/rest/SubmittedVariantsRestController.java

```diff
@@ -93,5 +107,60 @@ public SubmittedVariantsRestController(
                         service.getLastInactive(identifier),
                         SubmittedVariant::new));
     }
-}
 
+    @GetMapping(value = "/beacon/query", produces = "application/json")
+    public BeaconAlleleResponse doesVariantExist(@RequestParam(name="assemblyId") String assembly,
```

Comment:
The Beacon request only has 3 required fields `referenceName`, `referenceBases`, `assemblyId` and those will not be enough to calculate the hash.

Also for assemblyId we would need something like `GCA_000001405.1` but what the Beacon query will give us is 
>Assembly identifier (GRC notation, e.g. GRCh37).

Reply:
This deviation will be documented and sent to the beacon working group